### ISSUE TITLE
Cadence v0.11.0 changed sql.driverName to sql.pluginName 

### ISSUE THREAD

**Issue by martinxsliu** [banzaicloud/banzai-charts#1068]
**Describe the bug**
The latest (v0.11.0) version of Cadence changed their SQL configuration to use `pluginName` instead of `driverName` to differentiate between SQL providers (e.g. MySQL, Postgres). For example, see https://github.com/uber/cadence/pull/2909 and [code](https://github.com/uber/cadence/blob/d49eec3be4e7958de65504123c03b4bfc309c44b/common/service/config/config.go#L202). Afaict the functionality and behaviour are all the same, just the config key was changed.

**Steps to reproduce the issue:**
Upgrading server version (by setting the Helm `server.image.tag` value) to `0.11.0` will cause pods to crash loop with the error log:
```
2020/03/11 16:51:40 Config file corrupted.Persistence.DataStores[default](value).SQL.PluginName: zero value
```

**Additional context**
In the chart, the `driverName` is provided in the config map here: https://github.com/banzaicloud/banzai-charts/blob/master/cadence/templates/server-configmap.yaml#L35 and similarly for the visibility DB below. Those values are derived from the value keys `server.config.persistence.default.sql.driver` and `server.config.persistence.visibility.sql.driver`.

I'm happy to go in and add a new `plugin` field in the `server.config.persistence.x.sql` object, and wire it up to the config map, if it is okay.

**Comment by martinxsliu** [comment]
I just noticed that the config map will include any extra keys set by values.yaml [here](https://github.com/banzaicloud/banzai-charts/blob/master/cadence/templates/server-configmap.yaml#L41-L43), so I can just set the `server.config.persistence.default.sql.pluginName` key. I confirmed that it works.

**Comment by pregnor** [comment]
Mostly fixed in #1119 (`Upgrade Cadence to 0.11.0`), first released in chart [cadence/0.9.0](https://github.com/banzaicloud/banzai-charts/releases/tag/cadence%2F0.9.0).

The `values.prod.yaml` file's references were missed during this renaming thus in case you want to use that particular configuration file with chart versions 0.9.0-0.12.1 and Cadence server versions 0.11.0+, please make sure to either rename the corresponding `values.prod.yaml: .server.config.persistence.[].sql.driver` fields to `... .pluginName` or provide the `pluginName` key and value as an extra field as described in the workaround above.

Fixed in #1222 (`Upgraded Cadence server (0.14.1->)0.15.1, web (3.18.1->)3.22.2, chart (0.12.1->)0.13.0`), first released in chart [cadence/0.13.0](https://github.com/banzaicloud/banzai-charts/releases/tag/cadence%2F0.13.0).
